Templates/Databases/MicrosoftSQL/Query Microsoft SQL Database and return JSON GET/long-description.md

```diff
@@ -0,0 +1,54 @@
+This template process receives an **HTTP request**, queries data from a **MicrosoftSQL database** based on parameters given in the HTTP request, and returns the queried data to the caller as **JSON**. 
+
+![Template](assets/Query_Microsoft_SQL_Database_and_return_JSON_GET.svg)
+Templates\Databases\MicrosoftSQL\Query Microsoft SQL Database and return JSON GET\assets\Query_Microsoft_SQL_Database_and_return_JSON_GET.svg
```

Comment:
This should be a relative path, so just `assets/mypic.svg` should be fine. Also use forward slash please :)